package.json

```diff
@@ -12,7 +12,7 @@
     "test": "npm run test:contracts",
     "test:contracts": "truffle test",
     "test:integration": "cd test_integration && tsc && ./main.sh",
-    "test:dev": "truffle test test/anchor/anchor_state_root.js",
+    "test:dev": "truffle test test/origin_observer/setup.js",
```

Comment:
This change is not needed.

Reply:
True, its just dev thing, it keeps changing.